react/components/FormattedPrice.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FunctionComponent } from 'react'
```

Comment:
Let's use `FormattedCurrency` from [vtex-apps/format-currency](https://github.com/vtex-apps/format-currency) instead of implementing our own component. It gets the culture info from the store config so you don't need to pass the `currency` prop to it.